files/image_config/monit/memory_checker

```diff
@@ -54,6 +58,14 @@ def get_command_result(command):
 
     return command_stdout.strip()
 
+def publish_events(container_name, mem_usage_bytes, threshold_value):
+    events_handle = swsscommon.events_init_publisher(EVENTS_PUBLISHER_SOURCE)
```

Comment:
Why not we do this at the start of the process, which helps saves time, in waiting for async connect.
The init-publisher is not expensive
You can refer disk_check.py

Reply:
Done